### PR TITLE
[YUNIKORN-2786] Improve convert2Namespace funtion's test coverage

### PR DESCRIPTION
**Pull request by SP12893678**
### What is this PR for?
Improve convert2Namespace funtion's test coverage

### What type of PR is it?
* [x] -Test

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2786

### CODE REVIEW

File: pkg/admission/util_test.go [hunk 1]
```diff
@@ -189,3 +190,18 @@ func TestDefaultQueueName(t *testing.T) {
 		t.Fatal("UpdatePodLabelForAdmissionController is not as expected")
 	}
 }
+
+func TestConvert2Namespace(t *testing.T) {
+	configMap := &v1.Namespace{}
+	result := convert2Namespace(configMap)
+	assert.Equal(t, result != nil, true)
+	assert.Equal(t, reflect.DeepEqual(result, configMap), true)
```
Comment (brandboat) [inline pkg/admission/util_test.go:198]:
```suggestion
	assert.Check(t, reflect.DeepEqual(result, configMap))
```

File: pkg/admission/util_test.go [hunk 2]
```diff
@@ -189,3 +190,18 @@ func TestDefaultQueueName(t *testing.T) {
 		t.Fatal("UpdatePodLabelForAdmissionController is not as expected")
 	}
 }
+
+func TestConvert2Namespace(t *testing.T) {
+	configMap := &v1.Namespace{}
+	result := convert2Namespace(configMap)
+	assert.Equal(t, result != nil, true)
```
Comment (brandboat) [inline pkg/admission/util_test.go:197]:
```suggestion
	assert.Check(t, result != nil)
```

File: pkg/admission/util_test.go [hunk 3]
```diff
@@ -189,3 +190,18 @@ func TestDefaultQueueName(t *testing.T) {
...
+	assert.Equal(t, reflect.DeepEqual(result, configMap), true)
+
+	obj := struct{}{}
+	result = convert2Namespace(obj)
+	assert.Equal(t, result == nil, true)
```
Comment (brandboat) [inline pkg/admission/util_test.go:202]:
```suggestion
	assert.Check(t, result == nil)
```

File: pkg/admission/util_test.go [hunk 4]
```diff
@@ -189,3 +190,18 @@ func TestDefaultQueueName(t *testing.T) {
...
+
+	pod := &v1.Pod{}
+	result = convert2Namespace(pod)
+	assert.Equal(t, result == nil, true)
```
Comment (brandboat) [inline pkg/admission/util_test.go:206]:
```suggestion
	assert.Check(t, result == nil)
```

File: pkg/admission/util_test.go [hunk 5]
```diff
@@ -189,3 +190,18 @@ func TestDefaultQueueName(t *testing.T) {
 		t.Fatal("UpdatePodLabelForAdmissionController is not as expected")
 	}
 }
+
+func TestConvert2Namespace(t *testing.T) {
+	configMap := &v1.Namespace{}
```
Comment (brandboat) [inline pkg/admission/util_test.go:195]: could we rename to namespace ?

File: pkg/admission/util_test.go (hunk 2 above)
Comment (brandboat) [inline pkg/admission/util_test.go:197]:
```suggestion
	assert.Check(t, result != nil)
```

**Review by brandboat — COMMENTED** [review]
Thanks for the patch, overall lgtm, just some nits.

**Comment by SP12893678** [comment]
> Thanks for the patch, overall lgtm, just some nits.

Thanks for your review!
I have fixed it

**Review by chenyulin0719 — APPROVED** [review]
LGTM

**Review by ryankert01 — APPROVED** [review]
+1, LGTM

**Pull request closed without merging**